src/client/pages/OfferNew/utils.ts

```diff
@@ -262,13 +262,12 @@ export const ssnLengthByMarket: Record<Market, number> = {
 export const createSsnValidator = (market: Market) => (
   ssn: string,
 ): boolean => {
-  if (market === Market.No) {
-    return /^[0-9]{2}[0,1][0-9][0-9]{2}[ ]?[0-9]{5}$/.test(ssn)
-  } else if (market === Market.Dk) {
-    return /^[0-9]{2}[0,1][0-9][0-9]{2}[ ]?[0-9]{4}$/.test(ssn)
-  } else if (market === Market.Se) {
-    return /^([1-2][0-9])?[0-9]{2}[0-1][0-9][0-9]{2}[-+]?[0-9]{4}$/.test(ssn)
+  switch (market) {
+    case Market.No:
+      return /^[0-9]{2}[0,1][0-9][0-9]{2}[ ]?[0-9]{5}$/.test(ssn)
+    case Market.Dk:
+      return /^[0-9]{2}[0,1][0-9][0-9]{2}[ ]?[0-9]{4}$/.test(ssn)
+    case Market.Se:
+      return /^([1-2][0-9])?[0-9]{2}[0-1][0-9][0-9]{2}[-+]?[0-9]{4}$/.test(ssn)
   }
-
-  throw new Error(`Expected market to be NO, DK or SE but was ${market}`)
```

Comment:
You could throw the error in the default case but, maybe were not planning to add any new markets any time soon and update the enum.

Reply:
The beauty of typescript is that `switch`es become exhausitive, meaning that whenever we add a market to the enum this'll give a compile time type error which imo is a much better way of detecting errors. So i _think_ this should never happen